### PR TITLE
fix(chip): update padding for transparent variant (#7164)

### PR DESCRIPTION
**Pull request by muraliSingh7**
This PR addresses the issue described in #7164, where the Chip component's padding for the Transparent variant was not correctly aligned. The following changes have been made:

The padding-left for the Transparent variant is now set to 2px to improve alignment.
Other chip variants retain the default horizontal padding values.

These changes ensure that the Transparent chip variant is rendered consistently with the rest of the design, as described in #7164.

See screenshots below for before and after comparison:
Before:
image
image

After:
Screenshot 2024-12-01 at 4 02 08 AM
image

Please review @Bonapara and let me know if you have any feedback or concerns.

### CODE REVIEW

**Comment by Bonapara** [comment]
Thanks for the PR @muraliSingh7!

On Figma, the padding-left of the table cells should be 8px. It looks like it's 6px today. Can you fix that at the same time?

### Desired

CleanShot 2024-12-02 at 09 23 16

### Current

CleanShot 2024-12-02 at 09 25 11

**Comment by muraliSingh7** [comment]
@Bonapara I think I don't have access to edit the Figma files. However, I have made some styling changes directly in the code, which changes the padding to 6px.

**Comment by Bonapara** [comment]
I think we misunderstood each other, @muraliSingh7. The padding-left is currently 6px and should be changed to 8px (Figma). Thanks!

**Comment by muraliSingh7** [comment]
@Bonapara I don't have the necessary permissions to edit Figma files. You may reach out to your team members for that. Could you please tag someone from the UI development team to review this UI code fix ?

**Comment by Bonapara** [comment]
I think we confused each other @muraliSingh7. It should be changed to 8px in the code, not in Figma.

**Comment by muraliSingh7** [comment]
@Bonapara  now I understand you want 8px padding-left across all cells. Am I right now?
I have modified styling in two files I hope this fix the issue.

**Comment by Bonapara** [comment]
That's it! Thanks @muraliSingh7

File: packages/twenty-ui/src/display/chip/components/Chip.tsx [hunk 1]
```diff
@@ -116,6 +116,12 @@ const StyledContainer = withTheme(styled.div<
   & > svg {
     flex-shrink: 0;
   }
+
+  padding-left: ${({ variant }) =>
+    variant === ChipVariant.Transparent
+      ? '0px'
```
Comment (Bonapara) [inline packages/twenty-ui/src/display/chip/components/Chip.tsx:122]: Maybe we should use theme.spacing(0);

Comment (muraliSingh7) [reply packages/twenty-ui/src/display/chip/components/Chip.tsx]: Yeah I updated it to theme.spacing(0)

**Comment by guillim** [comment]
LGTM. Thanks for your code @muraliSingh7 !!! 🙏

**Review by guillim — APPROVED** [review]

**Pull request merged**